ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/PTP_TESTS_LEAFS.yml

```diff
@@ -9,55 +9,58 @@ l3leaf:
     uplink_switches: ['ptp-tests-spine1', 'ptp-tests-spine1']
     uplink_ipv4_pool: 10.254.2.0/24
     vtep_loopback_ipv4_pool: 10.254.11.0/27
-    mlag_interfaces: ['Ethernet11', 'Ethernet12']
+    mlag_interfaces: ['Ethernet9', 'Ethernet10']
     mlag_peer_ipv4_pool: 10.254.1.64/27
     mlag_peer_l3_ipv4_pool: 10.254.1.96/27
     spanning_tree_priority: 4096
     spanning_tree_mode: mstp
     ptp:
       enabled: true
+      mlag: true
```

Comment:
do we still test "somewhere" that the ptp config is not rendered on mlag when this setting is unset/false?

Reply:
I don't think so. If we need that, I need to add two more devices.